specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-02-01/managedClusters.json

```diff
@@ -1165,7 +1165,7 @@
         "name": "AgentPoolType",
         "modelAsString": true
       },
-      "description": "AgentPoolType represents types of an agent pool"
+      "description": "AgentPoolType represents types of an agent pool. VirtualMachineScaleSets type is still in PREVIEW."
```

Comment:
The correlation between agentpooltype and vmss isn't super clear - is there somewhere we state type = vmss is a pre-requisite for agentpool? We'd probably want to state it for AZ as well.

Reply:
marking vmss as required for az. For multi-agentpool, will include the change when it is ready in the next release.